test/e2e/storage/vsphere_volume_vsan_policy.go

```diff
@@ -204,7 +268,7 @@ func invokeValidVSANPolicyTest(f *framework.Framework, client clientset.Interfac
 	Expect(err).NotTo(HaveOccurred())
 	defer framework.DeletePersistentVolumeClaim(client, pvclaim.Name, namespace)
 
-	pvclaims := make([]*v1.PersistentVolumeClaim, 1)
+	pvclaims := make([]*v1.PersistentVolumeClaim, 0)
```

Comment:
I only need pvclaims of size 1.  The following code pvclaims := make([]*v1.PersistentVolumeClaim, 0) does that. 
Having make([]*v1.PersistentVolumeClaim, 1), creates a pvclaims of size 2 which is not needed.